auth/forward.go

```diff
@@ -0,0 +1,88 @@
+package auth
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/containous/traefik/log"
+	"github.com/containous/traefik/types"
+	"github.com/stretchr/stew/objects"
+)
+
+// Forward the authentication to a external server
+func Forward(forward *types.Forward, w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
+
+	client := http.Client{}
+
+	forwardReq, err := http.NewRequest("GET", forward.Address, nil)
+	if err != nil {
+		log.Debugf("Error calling %s. Cause %s", forward.Address, err)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	forwardReq.Header.Add("Accept", "application/json")
```

Comment:
@diegooliveira I see, I must have misunderstood the PR. I thought it forwards the original request as discussed in your PR.